integration/generic-tests/universalpayload_amd64_test.go

```diff
@@ -0,0 +1,99 @@
+// Copyright 2025 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build amd64 && !race
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/Netflix/go-expect"
+	"github.com/hugelgupf/vmtest/qemu"
+	"github.com/hugelgupf/vmtest/scriptvm"
+	"github.com/u-root/mkuimage/uimage"
+)
+
+// TestUPLBootAmd64 tests '/bbin/kexec UplFitX64.fit' to boot to UEFI Shell.
+func TestUPLBootAmd64(t *testing.T) {
+	//os.Setenv("VMTEST_ARCH", "amd64")
+	qemu.SkipIfNotArch(t, qemu.ArchAMD64)
+
+	// Check required images, including:
+	//   OVMF.fd 		-- UEFI OVMF binary
+	//   bzImage 		-- Linuxboot kernel image
+	//   UplFitX64.fit	-- UPL Amd64 image with FDT enabled
+
+	var ovmf string
+	var bzImage string
+	var upl string
+
+	if img := os.Getenv("VMTEST_OVMF"); len(img) == 0 {
+		t.Skipf("VMTEST_OVMF not set!!")
+	} else {
+		ovmf = img
+	}
+
+	if _, err := os.Stat(ovmf); err != nil && os.IsNotExist(err) {
+		t.Skipf("OVMF.fd image is not found: %s\n", ovmf)
+	}
+
+	if img := os.Getenv("VMTEST_KERNEL"); len(img) == 0 {
+		t.Skipf("VMTEST_KERNEL not set!!")
+	} else {
+		bzImage = img
+	}
+
+	if _, err := os.Stat(bzImage); err != nil && os.IsNotExist(err) {
+		t.Skipf("Linux kernel bzImage image is not found: %s\n", bzImage)
+	}
+
+	if img := os.Getenv("UROOT_TEST_UPLFIT"); len(img) == 0 {
+		t.Skipf("UROOT_TEST_UPLFIT not set!!")
```

Comment:
Does it make sense to add something like a standard config somewhere (not sure where though), similar to the one [provided by `vmtest`](https://github.com/hugelgupf/vmtest/blob/5d9f3d34a58dc7b13bca786e8ac32d3e2ce4e95d/tools/runvmtest/main.go#L74) to pick up the files supplied by your Dockerfiles? Or am I overseeing how the integration tests in the CI pick up the right environment variables?

Reply:
Sure. 
Let me separate this PR to different PRs:
1. One PR to update docker images.
2. One PR to add environment variable in vmtest and .vmtest.ymal to get docker images from ghcr.io